conformance_tests/sysman/test_sysman_temperature/src/test_sysman_temperature.cpp

```diff
@@ -25,6 +25,9 @@ class TempModuleTest : public lzt::SysmanCtsClass {};
 #define TEMPERATURE_TEST TempModuleTest
 #endif // USE_ZESINIT
 
+constexpr double invalid_min_temperature = 10;
```

Comment:
nit: For variable names, how about "min_valid_temperature" and "max_valid_temperature"?
These names emphasize that the values represent the valid range for temperature, making the code easier to read.

Reply:
min_valid_temp and max_valid_temp would be sufficient, thank you

Reply:
![Screenshot 2024-09-24 140810](https://github.com/user-attachments/assets/9a08e1e0-0c1d-4e2e-94f8-aeaaf2463851)

Reply:
This invalidMin/MaxTemperature terminology is used in compute-runtime repo, so this terminology was given in my jira.

Please confirm if this needs to be changed to min_valid_temp format or the same format can be retained.

Reply:
Like I said, it's a 'nit'. It's just a suggestion which could help increase the code readability(in my view).
I suggest you to go with your intuition and choose the variables names.

Either way PR is good to merge.

Reply:
I have changed it to max_valid_temperature and min_valid_temperature as suggested. But now do I have to change the min_valid_temperature to 11 from 10, and max_valid_temperature to124 from 125 ? since 10 and 125 are invalid minimum and maximum temperatures respectively according to sysman_const.h file

Please confirm.

Reply:
Yes, you can update to 11 and 124 and update the temperature checks to GE & LE instead of GT & LT

Reply:
Actual check from one of the driver file https://github.com/intel-innersource/drivers.gpu.compute.runtime/blob/d5e12432df8c2e1f7076cb22502ac4176f63b2b8/level_zero/sysman/source/shared/linux/product_helper/sysman_product_helper_hw.inl#L161  is as follows

`if ((temperature > invalidMaxTemperature) || (temperature < invalidMinTemperature)) {return false;}`

This means valid range is [10, 125], both values inclusive.
So use EXPECT_GE and EXPECT_LE with these values.